core/src/io/anuke/mindustry/content/Blocks.java

```diff
@@ -1832,6 +1832,14 @@ public void draw(Bullet b){
             consumes.power(0.05f);
         }};
 
+        largeIlluminator = new LightBlock("large-illuminator"){{
+            requirements(Category.effect, BuildVisibility.lightingOnly, ItemStack.with(Items.graphite, 8, Items.silicon, 4));
+            brightness = 1.34f;
```

Comment:
Brightness is the light opacity; it can't and shouldn't be greater than 1.

Reply:
I'll play around with this a bit, thanks for mentioning this. If a lack of better option arises, I'll leave it at the value from the standard illuminator.